README.md

```diff
@@ -1,70 +1,91 @@
-# Getting Started with Create React App
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/maosan132/calculator/">
+    <img src="./mLogo.png" alt="Logo" width="80" height="80">
+  </a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+  <h3 align="center">Calculator -
+  <a href='https://maosan132.github.io/calculator/'> Live version</a></h3>
 
-## Available Scripts
+  <p align="center">
+    This project is part of the Microverse React Section!
+    <br />
+    <a href="https://github.com/maosan132/calculator/">Report Bug</a>
+    ·
+    <a href="https://github.com/maosan132/calculator/">Request Feature</a>
+  </p>
+</p>
 
-In the project directory, you can run:
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
 
-### `npm start`
+* [About the Project](#about-the-project)
+  * [Installation](#installation)
+  * [Built With](#built-with)
+* [Contact](#contact)
+* [Acknowledgements](#acknowledgements)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+<!-- ABOUT THE PROJECT -->
+## About The Project
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+This purpose of this project is to build a simple product that would assist us with our calculations, that is just fully-functional and it does not need to be beautiful and user-friendly (at least in its first version).
```

Comment:
- [ ] Please change `This purpose` to `The purpose`

Reply:
Done, thanks.